hdrreport/hdrreport.go

```diff
@@ -1,11 +1,22 @@
 package hdrreport
 
 import (
+	"encoding/json"
 	"fmt"
-	"github.com/codahale/hdrhistogram"
+	"log"
 	"os"
+
+	"github.com/codahale/hdrhistogram"
 )
 
+// Quantiles contains common latency quantiles (p50, p95, p999)
+type Quantiles struct {
+	Quantile50  int64 `json:"50"`
+	Quantile95  int64 `json:"95"`
+	Quantile99  int64 `json:"99"`
+	Quantile999 int64 `json:"999"`
```

Comment:
TIOLI -- I might label these as "p50", "p95", etc. to match the column headers.

Reply:
And I also now see that this serialization matches what strest-grpc does. Sorry for the noise -- haven't finished my coffee yet this morning.